vaadin-combo-box-flow-parent/vaadin-combo-box-flow/src/main/java/com/vaadin/flow/component/combobox/ComboBox.java

```diff
@@ -151,37 +117,13 @@ public void remove() {
      *
      * @param pageSize
      *            the amount of items to request at a time for lazy loading
-     * @see {@link #setPageSize(int)}
+     * @see #setPageSize(int)
      */
     public ComboBox(int pageSize) {
         super("value", null, String.class, ComboBox::presentationToModel,
                 ComboBox::modelToPresentation);
-
-        // Configure web component to use key property from the generated
-        // wrapper items for identification
-        getElement().setProperty("itemValuePath", "key");
-        getElement().setProperty("itemIdPath", "key");
-
         setPageSize(pageSize);
-
-        addAttachListener(e -> initConnector());
-
         setItems(new DataCommunicator.EmptyDataProvider<>());
```

Comment:
Moving this to `ComboBoxBase` results in `ComboboxSerializableTest` failing. Could not figure out to solve that, so it would need to be duplicated as well.